src/cpp/session/resources/tutorial_resources/index.html

```diff
@@ -7,9 +7,23 @@
    </head>
 
    <body class="rstudio-tutorials" style="visibility: hidden;">
+
       <div class="rstudio-tutorials-container">
          #!tutorials#
       </div>
+
+      <div class="rstudio-tutorials-footer">
+
+         <img class="rstudio-tutorials-logo" alt="RStudio Icon" src="./rstudio-logo.png">
```

Comment:
The icon doesn't add any useful information to a screen reader user, so just use an empty `alt` to mark it as purely a decorative element that a screen-reader can ignore.

`<img class="rstudio-tutorials-logo" alt src="./rstudio-logo.png">`